spec/page_number_spec.rb

```diff
@@ -23,12 +23,12 @@ def num
       (num.is_a? Numeric).should be
     end
 
-    it "is a kind of Fixnum" do
-      (num.is_a? Fixnum).should be
+    it "is a kind of Integer" do
+      (num.is_a? Integer).should be
     end
 
-    it "isn't directly a Fixnum" do
-      (num.instance_of? Fixnum).should_not be
+    it "isn't directly a Integer" do
+      (num.instance_of? Integer).should_not be
```

Comment:
This change weakens the assertion on Ruby 2.3 or earlier, since one could make `num` an instance of `Fixnum` and still make the test pass (`num` should be an instance of `PageNumber`).